xmake/modules/private/xrepo/action/install.lua

```diff
@@ -79,7 +79,8 @@ function menu_options()
                                        "    - xrepo install -p iphoneos -a arm64 \"zlib >=1.2.0\"",
                                        "    - xrepo install -p android [--ndk=/xxx] -m debug \"pcre2 10.x\"",
                                        "    - xrepo install -p mingw [--mingw=/xxx] -k shared zlib",
-                                       "    - xrepo install conan::zlib/1.2.11 vcpkg::zlib"}
+                                       "    - xrepo install conan::zlib/1.2.11 vcpkg::zlib",
+                                        values = function (complete, opt) return import("private.xrepo.quick_search.completion")(complete, opt) end}
```

Comment:
it will break `xrepo install -h`

> error: attempt to index a boolean value (local 'complete')

you need handle help menu and disable complete.

```lua
if complete then
    -- do complete
end
```

Reply:
I intercepted the help with "helpmenu" option in the completion funciton (in [b1d694c](https://github.com/xmake-io/xmake/pull/3214/commits/b1d694ca3cd1230d7845b3a7aad06c40b0c3c453))

Do you think it is enough ?

Reply:
Also, help menu should be able to handle the new kind of data.

In the values function for xrepo install, I return an array of objects with `value`, `description` and `versions` fields. I use value and description to build completion in json format (I put versions for an eventual later update of xrepo install completion).

For xrepo install help, it's useless to put values in help menu, but in other command completions like xrepo remove or xmake build for example, it could be interesting to change values function to also return name and description. Therefore, help menu have to handle those.

Reply:
> I intercepted the help with "helpmenu" option in the completion funciton (in [b1d694c](https://github.com/xmake-io/xmake/pull/3214/commits/b1d694ca3cd1230d7845b3a7aad06c40b0c3c453))
> 
> Do you think it is enough ?

sure